docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -1,27 +1,77 @@
+
+[role="screenshot"]
+image::images/additional-filters.png[Shows multiple ways to filter information]
+
+[float]
+[[customize-the-alerts-table]]
+=== Customize the Alerts table
+Use the icons in the upper left corner of the Alerts table to customize the columns you want displayed and to view the table in full screen mode.
+
+[role="screenshot"]
+image::images/alert-table-columns-and-size.gif[width=100%][height=100%][Demo that shows how to select the customize display icon and full screen icon]
+
+Click the *Customize Event Renderers* icon to enable event renderers within the Alerts table. When enabled, event renderers show relevant details that provide more context to the event. For example, if you enable the *Flow* Event Renderer, the Alerts table shows relevant details describing the flow of the data between a source and destination. These details could include hosts, ports, protocol, direction, duration, amount transferred, process, and geographic location.
+
+[role="screenshot"]
+image::images/customize-event-renderer.png[Shows the Event Renderer icon, 200]
+
+All event renderers are disabled by default. To switch between event views in the Alerts table, you can enable individual event renderers or click *Enable all*. Closing *Customize Event Renderers* page saves your configurations.
+
+[role="screenshot"]
+image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
+
+[float]
+[[view-alert-details]]
+=== View alert details
+To further inspect an alert, click the *View details* icon from the Alerts table.
+
+[role="screenshot"]
+image::images/view-alert-details.png[Shows the Event Renderer icon, 200]
+
+The Alert details flyout appears and offers several options for viewing alert details:
+
+* *Summary*: Offers an aggregated view of alert details. Alerts that have been enriched with `threat.indicator` data also display the *threat summary* section, which is an additional section located beneath the alert summary. In the *threat summary* section, you can view mapped data for the following `threat.indicator` subfields:
+** `matched.field`
+** `matched.type`
+** `source (threat.indicator.provider)`
+** `first_seen`
+** `last_seen`
+
+NOTE: If an alert has more than one threat, `threat.indicator` data is still aggregated under the *threat summary* section, but parsed out in the *Threat Intel* tab.
+
+* *Threat Intel*: Shows the number of matched intelligence sources and displays threats individually. Threats are organized by timestamp (the most recent threat alert is shown at the top and the oldest is at the bottom) and the available `threat.indicator` and `source.event` data is portrayed for each threat. If the alert has not been enriched with threat data the *Threat Intel* tab displays the message `No Threat Intel Enrichment Found` and provides a link to Threat Intel module documentation.
```

Comment:
we could consider updating this one as `Shows the number of matched threats and displays them individually`